driver-sync/src/test/functional/com/mongodb/client/AbstractCrudTest.java

```diff
@@ -170,7 +171,12 @@ private void runOperation(final BsonDocument expectedOutcome, final BsonDocument
             List<CommandEvent> expectedEvents = getExpectedEvents(definition.getArray("expectations"), databaseName, null);
             List<CommandEvent> events = commandListener.getCommandStartedEvents();
 
-
+            // There are tests where BulkWriteBatch creates two separate commands for updates and replaceOne
```

Comment:
Yuk. I think Isabella ran into this as well, and opened a SPEC ticket to change the spec tests so that they don't do these all in a single command.  Let's wait for those...

Reply:
I have incorporated the updated tests.